doc/dev/conda-builds.md

```diff
@@ -16,6 +16,20 @@ A Conda Artifact defines:
 
 ## How to Build an Azure SDK Conda Package Locally
 
+### Set up your conda environment
+
+You will notice that all the azure-sdk conda distributions have the **same** version number and requirement set. This is due to the fact that the azure-sdk team pushes our conda packages out in waves. To support this, all versions are set via a common environment variable `AZURESDK_CONDA_VERSION`.
+
+We keep this environment variable set properly across all our builds by using a common `environment.yml` when creating our build environment. This environment definition ensures that 
```

Comment:
Looks like a couple renames missed here.

Reply:
ctrl-f-ed to find the rest. Should have em all now. Thanks.